packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -143,6 +143,7 @@ class RefreshIndicator extends StatefulWidget {
     this.semanticsValue,
     this.strokeWidth = RefreshProgressIndicator.defaultStrokeWidth,
     this.triggerMode = RefreshIndicatorTriggerMode.onEdge,
+    this.elevation = 0,
```

Comment:
It looks like `RefreshProgressIndicator` has a default of `2` so this would be a minor breaking change as this would be different default behavior. Was there a reason to make it 0?

Reply:
Yes, elevation is missing refresh indicator api up until current flutter sdk. All the current user base will be affected as they might not need elevation. Hence keeping the elevation 0 will not break current ui functionality. 

That is what my thought process is.

@MitchellGoodwin What would you suggest? or this is fine? because i wrote the new test-cases and non of the existing test cases are failing

Reply:
They did actually have a hardcoded elevation before of 2, because nothing was provided to the `RefreshIndicator` it used it's default value of 2. https://github.com/flutter/flutter/blob/f2ba2049174d28b08d7466946eb4362bbec6b821/packages/flutter/lib/src/material/progress_indicator.dart#L881

So by having this provide a default of `0` we are changing the default elevation from 2 to 0. It also looks like the Material 2 default refresh indicator should have some elevation:
https://m2.material.io/design/platform-guidance/android-swipe-to-refresh.html#usage

I think it would be best to have a default of 2, and adjust the test accordingly.